src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -175,7 +176,7 @@ using LayerTypes = boost::variant<
     DropConnect<arma::mat, arma::mat>*,
     Dropout<arma::mat, arma::mat>*,
     AlphaDropout<arma::mat, arma::mat>*,
-    ELU<arma::mat, arma::mat>*,
+    // ELU<arma::mat, arma::mat>*,
```

Comment:
Can you comment the ``ReinforceNormal`` layer instead as it's mainly used the the glimpse network?